### PR TITLE
Updated readme with a more in-depth description of zkVerify

### PR DESCRIPTION
**Pull request by DanieleDiBenedetto**
… minor additions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,17 +9,36 @@ It is based on the [Substrate](https://substrate.io/) framework.
 > The plan for going live on Mainnet will be communicated later.
 > For more information see [zkVerify.io](https://zkverify.io/).
 
+## What is zkVerify
+zkVerify is a blockchain designed to provide zero knowledge proof verifications for any project or dApp using zero knowledge proofs. A high performance, public, decentralized, reliable, and secure blockchain, zkVerify has dedicated proof verification methods written in Rust, and available to be used in a modular and composable way.
```
Comment (giacgus) [inline README.md:13]:
```suggestion
zkVerify is a **L1** blockchain designed to provide verification for zero knowledge proofs.
as an high performance, public, decentralized, reliable, and secure blockchain, zkVerify has dedicated proof verification methods written in Rust, and available to be used in a modular and composable way.
```

File: README.md [hunk 2]
```diff
@@ -9,17 +9,36 @@ It is based on the [Substrate](https://substrate.io/) framework.
...
+
+## Motivation
+As execution and data availability have been offloaded from monolithic blockchain stacks, we believe that proof verification and settlement need specialization. This system provides such specialization and, by doing so, enables the next wave of innovation for ZK cryptography within Web3 and beyond.
```
Comment (giacgus) [inline README.md:16]:
```suggestion
As execution and data availability have been offloaded from monolithic blockchain stacks, we believe that proof verification and settlement need specialization. This system provides such enables the next wave of innovation for ZK cryptography within Web3 and beyond.
```

File: README.md [hunk 3]
```diff
@@ -102,15 +121,18 @@ The client will run a chain with a single validator (Alice) and start producing
...
 
 zkVerify includes some Docker files for building the client and running one or more nodes locally.
 For more information, see [docker/README.md](docker/README.md).
 
+We also provide Docker images to run a validator, boot or RPC node directly in the public testnet, via a user-friendly installation and deployment process. Please take a look at [How to run a node](https://docs.zkverify.io/tutorials/how_to_run_a_node/getting_started) section of the official documentation. 
+
+## Documentation
+
+The official documentation is available at [docs.zkverify.io](https://docs.zkverify.io/).
+
```
Comment (giacgus) [inline README.md:135]: Maybe add here reference to official Discord

**Review by zain-cheng — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (DanieleDiBenedetto) [reply README.md:135]: Doesn't seem the correct place here, besides it's referenced in the docs

File: README.md (hunk 1 above)
Comment (DanieleDiBenedetto) [reply README.md]: Suggestion applied

File: README.md (hunk 2 above)
Comment (DanieleDiBenedetto) [reply README.md]: Suggestion applied

**Review by giacgus — APPROVED** [review]

**Pull request merged**